src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/DeleteInstanceV35.sql

```diff
@@ -0,0 +1,264 @@
+/***************************************************************************************/
+-- STORED PROCEDURE
+--     DeleteInstanceV35
+--
+-- FIRST SCHEMA VERSION
+--     35
+--
+-- DESCRIPTION
+--     Removes the specified instance(s) and places them in the DeletedInstance table for later removal
+--
+-- PARAMETERS
+--     @partitionKey
+--         * The Partition key
+--     @cleanupAfter
+--         * The date time offset that the instance can be cleaned up.
+--     @createdStatus
+--         * Status value representing the created state.
+--     @studyInstanceUid
+--         * The study instance UID.
+--     @seriesInstanceUid
+--         * The series instance UID.
+--     @sopInstanceUid
+--         * The SOP instance UID.
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.DeleteInstanceV35
```

Comment:
I don't see any signature change from existing, why is the new version necessary?

Reply:
Yeah not needed. Will update it. Just went with a flow